server/controllers/clarifaiAPIs/index.js

```diff
@@ -1,16 +1,26 @@
-const apparelAPI = require('./apparelAPI');
-const colorAPI = require('./colorAPI');
-const generalAPI = require('./generalAPI');
+const apparelAPI = require("./apparelAPI");
+const colorAPI = require("./colorAPI");
+const generalAPI = require("./generalAPI");
 
 exports.clarifaiAPIs = (req, res) => {
   const { image_url } = req;
-  Promise.all([apparelAPI(image_url), colorAPI(image_url), generalAPI(image_url)]).then((values) => {
+  Promise.all([
+    apparelAPI(image_url),
+    colorAPI(image_url),
+    generalAPI(image_url)
+  ]).then(values => {
     const data = {
       image_url,
       apparel: values[0],
       colors: values[1],
-      general: values[2],
+      general: values[2]
     };
+    console.log("START HERE");
+    console.log(data.apparel.data);
+    console.log(data.colors.data);
+    console.log(data.general.data);
+    console.log("END HERE");
+
```

Comment:
this file should not be on this pull request if you want to keep it here remove all console.log() please